src/main/kotlin/com/trivigenai/plugins/Routing.kt

```diff
@@ -0,0 +1,36 @@
+package com.trivigenai.plugins
```

Comment:
Isn't better to delete the getEndpoints() function and put it's code to configureRouting()? There is nothing apart from this function, so it can be moved.